.github/workflows/build.yml

```diff
@@ -48,24 +48,20 @@ jobs:
     - name: Generate configure script
       run:
         ./build.sh
-    - name: configure
+    - name: Run configure script
       run: ./configure $CONFIGURE_FLAGS
-    - name: Make
-      run: make
-    - name: make check
-      continue-on-error: true
-      run: make check
+    - name: Build collectd
+      run: make $MAKEFLAGS
+    - name: Run make check
+      run: make $MAKEFLAGS check
     - name: Dump test logs
       run: |
           cat ./test-suite.log || true
-    - name: install bzip2
-      continue-on-error: true
-      run: |
+    - name: Install bzip2
```

Comment:
what about installing bzip2 in the used containers?

Reply:
That's a possibility. It'll take a while to update all the containers though, the changes are trivial, but the process is tedious)
But it should save a little bit of time in the CI runs.

I can update this PR to remove these, and then it'll become dependant on the new container changes.